doc/source/user_guide/timeseries.rst

```diff
@@ -677,6 +677,7 @@ If index resolution is second, then the minute-accurate timestamp gives a
 If the timestamp string is treated as a slice, it can be used to index ``DataFrame`` with ``[]`` as well.
 
 .. ipython:: python
+   :okwarning:
 
     dft_minute = pd.DataFrame({'a': [1, 2, 3], 'b': [4, 5, 6]},
```

Comment:
I think the whitespace alignment is off here (`:okwarning:` and `dft_minute...` should be vertically aligned)